app/views/admin/enterprises/form/_shop_preferences.html.haml

```diff
@@ -77,16 +77,16 @@
       = f.radio_button :allow_order_changes, true, "ng-model" => "Enterprise.allow_order_changes", "ng-value" => "true"
       = f.label :allow_order_changes, t('.allow_order_changes_true'), value: :true
 
-
--# .row
--#   .alpha.eleven.columns
--#     .three.columns.alpha
--#       %label= t '.enable_subscriptions'
--#       %div{'ofn-with-tip' => t('.enable_subscriptions_tip')}
--#         %a= t 'admin.whats_this'
--#     .three.columns
--#       = f.radio_button :enable_subscriptions, true
--#       = f.label :enable_subscriptions, t('.enable_subscriptions_true'), value: :true
--#     .five.columns.omega
--#       = f.radio_button :enable_subscriptions, false
--#       = f.label :enable_subscriptions, t('.enable_subscriptions_false'), value: :false
+- if spree_current_user.admin?
```

Comment:
Is it too much to ask a `superadmin?` alias method? Please :pray:. It will make it much simpler to understand next time I read it.

Reply:
Sure, there are LOTS of uses of this method, if we are going to add a helper method, I feel like I should do them all. In which case it probably becomes a separate PR.